### PR TITLE
Added link to new openstack boxes and deleted all unused strings.

### PR DESCRIPTION
**Pull request by yuvb**

### CODE REVIEW

**Comment by MirantisWorkloadMobility** [comment]
There were no python files updated. Admins, please, check pull request.

File: cloudferry_devlab/Vagrantfile [hunk 1]
```diff
@@ -1,209 +1,161 @@
...
-    config.proxy.http     = "#{options['http_proxy']}"
-    config.proxy.https    = "#{options['https_proxy']}"
-    config.proxy.ftp      = "#{options['ftp_proxy']}"
-    config.proxy.no_proxy = "localhost,127.0.0.1,"\
-                            "#{options['grizzly_ip']},"\
-                            "#{options['icehouse_ip']},"\
-                            "#{options['juno_ip']},"\
-                            "#{options['cloudferry_ip']},"\
-                            "#{options['nfs_ip']}"
-  end
```
Comment (roman-verchikov) [inline cloudferry_devlab/Vagrantfile]: Let's keep support for proxy settings

File: cloudferry_devlab/Vagrantfile [hunk 2]
```diff
@@ -1,209 +1,161 @@
...
+    'role' => 'openstack',
+    'cpus' => 2
   },
-  "cloudferry" => {
-    "box" => "hashicorp/precise64",
-    "ip" => "#{options['cloudferry_ip']}",
-    "ip2" => "#{options['cloudferry_ip2']}",
-    "memory" => 2048,
-    "role" => "lab"
+  'cloudferry' => {
```
Comment (roman-verchikov) [inline cloudferry_devlab/Vagrantfile]: Let's remove 'cloudferry' VM, it's not needed

File: cloudferry_devlab/Vagrantfile [hunk 3]
```diff
@@ -1,209 +1,161 @@
...
-              source: "~/.ssh/id_rsa",
-              destination: "/home/vagrant/.ssh/id_rsa"
-          end
+      thisnode.vm.hostname = nodedata.fetch('hostname', nodename)
+      thisnode.vm.provision 'shell', inline: "echo '#{etc_hosts}' >> /etc/hosts"
+      thisnode.vm.provision 'shell',
+                            path: './provision/keys.sh',
+                            args: ['--public-key', File.read("#{ENV['HOME']}/#{options['public_key_path']}").strip]
+
+      if role == 'lab'
```
Comment (roman-verchikov) [inline cloudferry_devlab/Vagrantfile]: This part can be removed with removal of 'cloudferry' VM

**Comment by MirantisWorkloadMobility** [comment]
There were no python files updated. Admins, please, check pull request.

**Comment by MirantisWorkloadMobility** [comment]
There were no python files updated. Admins, please, check pull request.

**Comment by MirantisWorkloadMobility** [comment]
There were no python files updated. Admins, please, check pull request.

**Pull request closed without merging**